style.css

```diff
@@ -0,0 +1,610 @@
+  position: fixed;
+  background-color: #5965e9;
+  width: 60px;
+  height: 60px;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+  top: 200px;
+  display: flex;
+  align-items: center;
+  z-index: 99;
+}
+
+.darkmode_icon {
+  width: 25px;
+  margin-left: 22px;
+}
+
+.header {
+  position: fixed;
+  display: flex;
+  width: 1320px;
+  justify-content: flex-end;
+  z-index: 99;
+}
+
+.header {
+ padding-bottom: 5px;
+}
+
+#header_logo {
+  margin: 10px 130px 0px 10px;
+}
+
+#header_logo_dark {
+  margin: 10px 130px 0px 10px;
+}
+
+.digital {
+  width: 140px;
+}
+
+.header_item {
+  margin: 15px 20px 15px 20px;
+  font-weight: 500;
+  font-size: 15px;
+  display: flex;
+  align-items: center;
+}
+
+.header_item_arrow {
+  width: 20px;
+  height: 19px;
+  margin: 3px 0px 0px 1px;
+}
+
+.header_menu {
+  margin: 15px 20px 15px 40px;
+  width: 40px;
+  height: 40px;
+  background-color: #ecf2f6;
+  border-radius: 20px;
+}
+
+.header_menu_icon {
+  width: 18px;
+  margin: 11px
+}
+
+.line4 {
+  height: 350px;
+  position: absolute;
+  top: 520px;
+  z-index: 10;
+}
+
+.bubble9 {
+  position: absolute;
+  top: 340px;
+}
+
+.container_about {
+  margin-top: 150px;
+  display: flex;
+  position: relative;
+}
+
+h7 {
+  color: #d27171;
+  font-size: 13px;
+  font-weight: 600;
+}
+
+.aboutUs_title {
+  font-size: 30px;
+  margin: 10px 0px 15px 0px;
+  font-weight: 500;
+}
+
+.about_text {
+  width: 600px;
+  font-size: 13px;
+  margin-bottom: 35px;
+  color: #b0aeae
+}
+
+.list_select {
+  width: 600px;
+  font-weight: 500;
+  border-bottom: 2px solid rgba(81, 81, 81, 0.1);
+  padding-bottom: 10px;
+  color: #5957e9;
+}
+
+.list {
+  width: 600px;
+  font-weight: 500;
+  border-bottom: 2px solid rgba(81, 81, 81, 0.1);
+  padding-bottom: 10px;
+  margin-top: 10px;
+}
+
+.list_text {
+  padding-top: 8px;
+  font-weight: 400;
+  font-size: 11px;
+  color: #b0aeae;
+  align-items: center;
+}
+
+.earth {
+  display: inline;
+  width: 15px;
+  margin-right: 7px;
+}
+
+.strategy {
+  display: inline;
+  width: 17px;
+  margin-right: 4px;
+}
+
+.design {
+  display: inline;
+  width: 15px;
+  margin-right: 7px;
+}
+
+.left {
+  background-color: #fff;
+  width: 60%;
+  padding: 10px 0px 0px 65px;
+}
+
+.right {
+  background-color: #fff;
+  width: 40%;
+  padding: 15px;
+}
+
+.form_container {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 15px;
+  box-shadow: 0px 0px 10px 3px rgba(0, 0, 0, 0.05);
+}
+
+.form_title {
+  font-size: 28px;
+  font-weight: 600;
+  margin-bottom: 20px;
+}
+
+.form_name {
+  font-size: 12px;
+  font-weight: 800;
+  margin-bottom: 5px;
+}
+
+.form_inbox {
+  border: 1px solid rgba(0, 0, 0, 0.05);
+  border-radius: 7px;
+  padding: 7px;
+  margin-left: 5px;
+  box-shadow: 0px 0px 10px 3px rgba(0, 0, 0, 0.05);
+  margin-bottom: 15px;
+  color: rgba(0, 0, 0, 0.3);
+  font-size: 12px;
+}
+
+.form_button {
+  display: flex;
+  background-color: #5957e9;
+  justify-content: center;
+  height: 40px;
```

Comment:
Równie dobrze można to paddingiem zrobić bez użycia flexa i ustawiania wysokości.